app/views/root/_filter.html.erb

```diff
@@ -0,0 +1,81 @@
+<%
+  assignees = [{text: "All assignees", value: ""}]
+  content_types = []
+  states = []
+
+  @presenter.available_users.map do | user |
+    if user.id.to_s == @presenter.assigned_to
+      assignees << {text: user.name, value: user.id, selected: "true"}
+    else
+      assignees << {text: user.name, value: user.id}
+    end
+  end
+
+  format_filter_selection_options.map do | format |
+    if format[1] == @presenter.content_type
+      content_types << {text: format[0], value: format[1], selected: "true"}
+    else
+      content_types << {text: format[0], value: format[1]}
+    end
+  end
+
+  state_names.collect do |scope, status_label|
+    if @presenter.states.include? scope.to_s
+      states << {label: status_label, value: scope, checked: "true"}
+    else
+      states << {label: status_label, value: scope}
+    end
+  end
+%>
+
+<div class="publications-filter govuk-!-padding-5">
+  <%= form_with url: root_path, method: :get do |form| %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Filter by",
+      margin_bottom: 4,
+    } %>
+
+    <%= render "govuk_publishing_components/components/input", {
+      label: {
+        text: "Title",
+        bold: true,
+      },
+      name: "title_filter",
+      id: "title_filter",
+      value: @presenter.title,
+    } %>
+
+    <%= render "govuk_publishing_components/components/select", {
+      id: "assignee_filter",
+      full_width: true,
+      label: "Assigned to",
+      options: assignees,
+    } %>
+
+    <%= render "govuk_publishing_components/components/select", {
+      id: "format_filter",
```

Comment:
Maybe now would be a good time to rename the various usages of the term "format" with "content type" in the new (non-legacy) code.